streaming/base/storage/download.py

```diff
@@ -223,6 +254,10 @@ def download_file(remote: Optional[str], local: str, timeout: float):
     if os.path.exists(local):
         return
 
+    local = local.replace('\\', '/')
```

Comment:
Is this for windows file system ?

Reply:
Yes, os.path.join returns paths with `\` on Windows and remote file name e.g. `s3://test\example` doesn't work